src/libgcv/plugins/rhino/rhino_read.cpp

```diff
@@ -459,20 +459,72 @@ write_geometry(rt_wdb &wdb, const std::string &name, const ON_Geometry *geometry
     return 1;
 }
 
+std::string create_material(const ON_Material *im, rt_wdb& wdb)
+{
+    ON_Material dm;
+	
+    const ON_Material *material = (im) ? im : &dm;
```

Comment:
a locally scoped ON_Material probably isn't helpful here.  if "im" is null, then there's nothing to create and we probably shouldn't have called create_material().